src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/_consts.py

```diff
@@ -28,7 +28,8 @@
              'setting some default values.\n'
 MSG_CLOSING = '\nYou\'re all set! Here are some commands to try:\n'\
                ' $ az vm create --help\n'\
-               ' $ az feedback\n'
+               ' $ az feedback\n'\
+               ' $ az login\n'
```

Comment:
Consider moving `az login` to the top of the list as you can't create a vm without logging in.